### PR TITLE
Adds stand_threshold param to patchmax call

### PR DESCRIPTION
**Pull request by strangeties**
validated for url, http://localhost:8000/planscape-backend/forsys/generate_project_areas/single_scenario/?request_type=3&cluster_algorithm_type=0&priority_weights=5&priority_weights=1&priority_weights=1&priority_weights=1&priority_weights=1

image
image

### CODE REVIEW

File: src/planscape/forsys/generate_projects_for_a_single_scenario.R [hunk 1]
```diff
@@ -95,6 +96,7 @@ generate_projects_for_a_single_scenario <- function(
       patchmax_SDW = 1,
       patchmax_EPW = 0.5,
       patchmax_sample_frac = 0.01,
+      stand_threshold = paste(eligibility_field, ">0", sep=""),
```
Comment (JanLauGe) [inline src/planscape/forsys/generate_projects_for_a_single_scenario.R:99]: nit: Infix operators should be surrounded by spaces.
https://style.tidyverse.org/syntax.html#infix-operators

Could also use `paste0()`

**Review by JanLauGe — APPROVED** [review]
What's happening with the black pixels? They are new, right?

**Review by runforyourlives — APPROVED** [review]

**Comment by strangeties** [comment]
Yes: the black pixels are new.

Ptal at #655 for an longer explanation.

In short, the black pixels represent stands that are missing some condition scores. Up until #655, these stands were dropped. Now they're sent to Forsys with condition impact score = 0 so that they may still be added to project areas (with an untreatable stand penalty).

File: src/planscape/forsys/generate_projects_for_a_single_scenario.R (hunk 1 above)
Comment (strangeties) [reply src/planscape/forsys/generate_projects_for_a_single_scenario.R:99]: done.

File: src/planscape/forsys/views.py [hunk 2]
```diff
@@ -191,7 +191,9 @@ def run_forsys_generate_project_areas_for_a_single_scenario(
         forsys_input_dict: dict[str, list],
         forsys_proj_id_header: str, forsys_stand_id_header: str,
         forsys_area_header: str, forsys_cost_header: str,
-        forsys_geo_wkt_header: str, forsys_priority_headers: list[str], forsys_condition_headers: list[str],
+        forsys_geo_wkt_header: str, 
+        forsys_treatment_eligibility_header: str, 
+        forsys_priority_headers: list[str], forsys_condition_headers: list[str],
```
[the remark this answers is not in the document]
Comment (strangeties) [reply src/planscape/forsys/views.py:196]: done.
Reformatted the file.
I also opted to pass the entire ForsysInputHeaders object, which didn't originally make sense, but is much cleaner now.

**Pull request merged**